arviz/rcparams.py

```diff
@@ -93,11 +102,64 @@ def _validate_boolean(value):
     return value is True or value == "true"
 
 
+def iterable_validator(validate_scalar, length=None):
```

Comment:
Mainly for convention, I would name the function argument `scalar_validator` and the function itself `make_iterable_validator`.

Reply:
Alright, I'll make the changes.